tests/System.Azure.Experimental.Tests/System.Azure.Experimental.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <AssemblyOriginatorKeyFile>../../tools/test_key.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.3.1" />
```

Comment:
Do you expect this package to be used in other projects within the repo?

Yes, this is what we generally do for all packages coming from corefx (https://github.com/dotnet/corefxlab/search?utf8=%E2%9C%93&q=CoreFxVersion&type=) since it gives a single place to update the versions at once. It is set to 4.3.0 (in tools/dependencies).

If we want a specific newer version for ongoing work (for example System.Memory), we have specific properties for those individual packages that get updated whenever a new version get's pushed (via Maestro):
https://github.com/dotnet/corefxlab/blob/master/tools/dependencies.props#L5-L7

Although, the new packages aren't building (https://github.com/dotnet/corefxlab/pull/1613).

@shiftylogic, to confirm.